Add intro and installation topics (#22)
AdobeDocs/commerce-extensibility: src/pages/admin-ui-sdk/installation.md

---
title: Install Adobe Commerce Admin UI SDK
description: Learn how to install the Adobe Commerce Admin UI SDK.
---

# Install Adobe Commerce Admin UI SDK

> This topic describes the basic steps you need to install the Admin UI SDK.

Reviewer: ```suggestion
This topic describes the basic steps to install the Admin UI SDK.
```

## Prerequisites

### Adobe Commerce

> * Adobe Commerce on cloud infrastructure or on premises: 2.4.5+

Reviewer 2: Did we double check that the AdminIMS module is available for 2.4.5?

Author: IMS support is mentioned in the 2.4.5 release notes

https://experienceleague.adobe.com/docs/commerce-operations/release/notes/adobe-commerce/2-4-5.html?lang=en#adobe-commerce-integration-with-adobe-ims

* PHP 8.1+
* [Adobe Identity Management Service (IMS) for Adobe Commerce](https://experienceleague.adobe.com/docs/commerce-admin/start/admin/ims/adobe-ims-integration-overview.html)

<InlineAlert variant="info" slots="text1" />

Magento Open Source is not supported.

### Adobe App Builder

The [Adobe App Builder _Getting Started_ guide](https://developer.adobe.com/app-builder/docs/getting_started/) lists the latest software requirements for creating App Builder apps.

## Installation steps

You can install the Admin UI SDK on Adobe Commerce on cloud infrastructure and on-premises instances.

### Adobe Commerce on cloud infrastructure

> This method is used for installing the SDK for a cloud instance.

Reviewer: ```suggestion
This method installs the SDK on a cloud instance.
```
Not sure how the SDK works, so not sure if `on` is appropriate here.

1. On your local workstation, change to the Cloud project root directory.

1. Update your `composer.json` file:

   ```bash
   composer require magento/module-commerce-backend-uix": ">=1.0
   ```

1. Update dependencies and install the extension:

   ```bash
   composer update
   ```

   The `composer update` command updates all dependencies. If you do not want to update all dependencies at the same time, use this command instead:

   ```bash
   composer update magento/module-commerce-backend-uix
   ```

1. Commit and push your changes.

### On-premises

> This method is used for installing the SDK for an On-premises instance.

Reviewer: ```suggestion
This method installs the SDK on an On-premises instance.
```

1. Add the SDK module to the `require` section of the `composer.json` file:

   ```bash
   composer require magento/module-commerce-backend-uix": ">=1.0
   ```

1. Update dependencies and install the extension:

   ```bash
   composer update
   ```

   The `composer update` command updates all dependencies. If you do not want to update all dependencies at the same time, use this command instead:

   ```bash
   composer update magento/module-commerce-backend-uix
   ```

1. Upgrade Adobe Commerce:

   ```bash
   bin/magento setup:upgrade
   ```

1. Clear the cache:

   ```bash
   bin/magento cache:clean
   ```

> 1. Commit changes.

Reviewer: ```suggestion
1. Commit your changes.
```
